plugin.json

```
@@ -1,8 +1,11 @@
 {
     "id": "autotranslate",
     "name": "Autotranslate",
-    "description": "Autotranslate plugin for Mattermost 5.4 and later.",
-    "version": "0.2.0",
+    "description": "Autotranslate plugin for Mattermost 5.22 and later.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-autotranslate",
+    "support_url": "https://github.com/mattermost/mattermost-plugin-autotranslate/issues",
```

While touching this file let's also add:
```suggestion
    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-autotranslate/releases/tag/v0.3.0",
```